### PR TITLE
[DOC-11556] CV setting and metric scopes for v24.3

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-11556] CV setting and metric scopes for v24.3

Previews:
[src/current/v24.3/cluster-virtualization-metric-scopes.md](https://deploy-preview-19051--cockroachdb-docs.netlify.app/docs/v24.3/cluster-virtualization-metric-scopes.html)
[src/current/v24.3/cluster-virtualization-setting-scopes.md](https://deploy-preview-19051--cockroachdb-docs.netlify.app/docs/v24.3/cluster-virtualization-setting-scopes.html)

[Instructions for generating these](https://docs.google.com/spreadsheets/d/1HIalzAhwU0CEYzSuG2m1aXSJRpiIyQPJdt8SusHpJ_U/edit?gid=1642767093#gid=1642767093)

### CODE REVIEW

File: src/current/v24.3/cluster-virtualization-metric-scopes.md [hunk 1]
```diff
@@ -32,23 +32,48 @@ SERVER: n/a
...
+- `auth.scram.conn.latency`
 - `backup.last-failed-time.kms-inaccessible`
-- `build.timestamp`
 - `changefeed.admit_latency`
 - `changefeed.aggregator_progress`
 - `changefeed.backfill_count`
 - `changefeed.backfill_pending_ranges`
 - `changefeed.batch_reduction_count`
 - `changefeed.buffer_entries_mem.acquired`
+- `changefeed.buffer_entries_mem.acquired.aggregator`
```
Comment (mdlinville) [inline src/current/v24.3/cluster-virtualization-metric-scopes.md:48]: @kathancox FYI

File: src/current/v24.3/cluster-virtualization-metric-scopes.md [hunk 2]
```diff
@@ -597,40 +662,42 @@ SERVER: n/a
 - `logical_replication.commit_latency`
 - `logical_replication.events_dlqed`
 - `logical_replication.events_dlqed_age`
+- `logical_replication.events_dlqed_by_label`
```
Comment (mdlinville) [inline src/current/v24.3/cluster-virtualization-metric-scopes.md:665]: @kathancox FYI

File: src/current/v24.3/cluster-virtualization-metric-scopes.md [hunk 3]
```diff
@@ -1666,8 +1785,26 @@ SERVER: n/a
 - `storage.wal.failover.switch.count`
 - `storage.wal.failover.write_and_sync.latency`
 - `storage.wal.fsync.latency`
+- `storage.write-amplification`
```
Comment (mdlinville) [inline src/current/v24.3/cluster-virtualization-metric-scopes.md:1788]: @rmloveland FYI, and the block just below

File: src/current/v24.3/cluster-virtualization-setting-scopes.md [hunk 4]
```diff
@@ -109,16 +139,27 @@ system-visible: Can be set / modified only from the system virtual cluster, but
...
 - `server.identity_map.configuration`
+- `server.jwt_authentication.audience`
+- `server.jwt_authentication.claim`
+- `server.jwt_authentication.client.timeout`
+- `server.jwt_authentication.enabled`
+- `server.jwt_authentication.issuers.configuration (alias: server.jwt_authentication.issuers)`
+- `server.jwt_authentication.issuers.custom_ca`
+- `server.jwt_authentication.jwks`
+- `server.jwt_authentication.jwks_auto_fetch.enabled`
+- `server.ldap_authentication.client.tls_certificate`
```
Comment (mdlinville) [inline src/current/v24.3/cluster-virtualization-setting-scopes.md:150]: @mikeCRL FYI

File: src/current/v24.3/cluster-virtualization-setting-scopes.md [hunk 5]
```diff
@@ -291,11 +338,12 @@ system-visible: Can be set / modified only from the system virtual cluster, but
 
 - `admission.disk_bandwidth_tokens.elastic.enabled`
 - `admission.kv.enabled`
+- `physical_replication.consumer.minimum_flush_interval (alias: bulkio.stream_ingestion.minimum_flush_interval)`
```
Comment (mdlinville) [inline src/current/v24.3/cluster-virtualization-setting-scopes.md:341]:
```suggestion
- `bulkio.stream_ingestion.minimum_flush_interval`
- `physical_replication.consumer.minimum_flush_interval`
```

File: src/current/v24.3/cluster-virtualization-setting-scopes.md [hunk 6]
```diff
@@ -313,13 +363,13 @@ system-visible: Can be set / modified only from the system virtual cluster, but
...
 - `kvadmission.store.provisioned_bandwidth`
-- `physical_replication.consumer.minimum_flush_interval (alias: bulkio.stream_ingestion.minimum_flush_interval)`
+- `kvadmission.store.snapshot_ingest_bandwidth_control.enabled`
 - `server.consistency_check.max_rate`
 - `server.rangelog.ttl`
 - `server.shutdown.lease_transfer_iteration.timeout (alias: server.shutdown.lease_transfer_wait)`
 - `spanconfig.bounds.enabled`
-- `spanconfig.range_coalescing.application.enabled (alias: spanconfig.tenant_coalesce_adjacent.enabled)`
 - `spanconfig.range_coalescing.system.enabled (alias: spanconfig.storage_coalesce_adjacent.enabled)`
+- `spanconfig.range_coalescing.application.enabled (alias: spanconfig.tenant_coalesce_adjacent.enabled)`
```
Comment (mdlinville) [inline src/current/v24.3/cluster-virtualization-setting-scopes.md:372]:
```suggestion
- `spanconfig.range_coalescing.system.enabled`
- `spanconfig.range_coalescing.application.enabled`
- `spanconfig.storage_coalesce_adjacent.enabled`
- `spanconfig.tenant_coalesce_adjacent.enabled`
```

**Review by florence-crl — APPROVED** [review]
lgtm

**Comment by mdlinville** [comment]
TFTR!

File: src/current/v24.3/cluster-virtualization-metric-scopes.md (hunk 3 above)
Comment (rmloveland) [reply src/current/v24.3/cluster-virtualization-metric-scopes.md:1788]: oooh thanks @mdlinville - these `storeliveness` things are related to https://github.com/cockroachdb/docs/pull/18989

**Pull request merged**